components/network/weechat/Makefile

```diff
@@ -46,6 +47,7 @@ CONFIGURE_OPTIONS += --disable-ruby
 CONFIGURE_OPTIONS += --enable-man
 CONFIGURE_OPTIONS += --enable-tests
 CONFIGURE_ENV += CA_FILE="/etc/certs/ca-certificates.crt"
+CONFIGURE_ENV += PERL=$(PERL.5.24)
```

Comment:
Isn't `CONFIGURE_ENV += PERL=$(PERL)` enough?
Otherwise looks good.

Reply:
Yes, it builds fine with only CONFIGURE_ENV settings but I was irritated during gmake publish. Without explicitly setting PERL_VERSION pkgmogrify is being called with the default perl settings:
/usr/bin/pkgmogrify -D PERL_ARCH=i86pc-solaris-64int -D PERL_VERSION=5.22 ...

Setting PERL_VERSION changed that:
/usr/bin/pkgmogrify -D PERL_ARCH=i86pc-solaris-64int -D PERL_VERSION=5.24 ...

Reply:
I mean that `CONFIGURE_ENV += PERL=$(PERL)` should be enough